json-examples/external-ot-shelf.json

```diff
@@ -0,0 +1,1195 @@
+{
+  "ietf-network:networks": {
+    "network": [
+      {
+        "network-id": "WDM-Network-1",
+        "network-types": {
+          "ietf-te-topology:te-topology": {
+            "ietf-optical-impairment-topology:optical-impairment-topology": {}
+          }
+        },
+        "ietf-te-topology:te-topology-identifier": {
+          "topology-id": "WDM-Network-1"
+        },
+        "ietf-te-topology:te": {},
+        "ietf-optical-impairment-topology:otsi-group": [
+          {
+            "otsi-group-id": "Red OTSiG (Forward)",
+            "otsi": [
+              {
+                "otsi-carrier-id": 1
+              }
+            ]
+          },
+          {
+            "otsi-group-id": "Red OTSiG (Reverse)",
+            "otsi": [
+              {
+                "otsi-carrier-id": 1
+              }
+            ]
+          },
+          {
+            "otsi-group-id": "Green OTSiG (Forward)",
+            "otsi": [
+              {
+                "otsi-carrier-id": 1
+              },
+              {
+                "otsi-carrier-id": 2
+              }
+            ]
+          },
+          {
+            "otsi-group-id": "Green OTSiG (Reverse)",
+            "otsi": [
+              {
+                "otsi-carrier-id": 1
+              },
+              {
+                "otsi-carrier-id": 2
+              }
+            ]
+          }
+        ],
+        "node": [
+          {
+            "node-1": "WDM-TE-Node-1",
+            "ietf-te-topology:te-node-id": "192.0.2.1",
+            "ietf-te-topology:te": {},
+            "ietf-network-topology:termination-point": [
+              {
+                "tp-id": "1",
+                "ietf-te-topology:te-tp-id": 1,
+                "ietf-te-topology:te": {
+                  "inter-domain-plug-id": "AQ=="
+                }
+              },
+              {
+                "tp-id": "2",
+                "ietf-te-topology:te-tp-id": 2,
+                "ietf-te-topology:te": {
+                  "inter-domain-plug-id": "Ag=="
+                }
+              },
+              {
+                "tp-id": "3",
+                "ietf-te-topology:te-tp-id": 3,
+                "ietf-te-topology:te": {
+                  "inter-domain-plug-id": "Awo="
+                }
+              },
+              {
+                "tp-id": "23",
+                "ietf-te-topology:te-tp-id": 23
+              }
+            ],
+            "ietf-te-topology:tunnel-termination-point": [
```

Comment:
I have run an automated json vs yang validation tool and the example is not consistent with the yang:

I think this should be in the object "te", not outside

"ietf-te-topology:te": { 
   "te-node-attributes": {...},
   "tunnel-termination-point": [...]

I also think that "ietf-optical-impairment-topology:ttp-transceiver": should not be inside the "local-link-connectivities" container

Reply:
Hi Esther, you're right , the file has to be fixed as you suggested.